package.json

```diff
@@ -24,7 +24,7 @@
   ],
```

Comment:
would it be possible to release the alpha first so we can run acceptance tests and what not

Reply:
I did release the version already. I'm re-running all jobs to see if that fixes it. If not, I'll dig deeper, although the unit tests are passing locally for me

Reply:
I ran a fresh build for 1.2.0-alpha.2 and that one is working!